languages/elixir/exercises/concept/tuples/.meta/example.ex

```diff
@@ -0,0 +1,59 @@
+defmodule KitchenCalculator do
+
+  # Get the number component from the volume-unit pair '{:unit, volume}'
+
+  def get_volume(volume_pair), do: elem(volume_pair, 1)
+
+  # Convert to millilitre to another unit
+
+  def to_millilitre({:cup, cups}) do
+    {:millilitre, cups * 240}
+  end
+
+  def to_millilitre({:fluid_ounce, floz}) do
+    {:millilitre, floz * 29.57}
+  end
+
+  def to_millilitre({:teaspoon, teaspoons}) do
+    {:millilitre, teaspoons * 5}
+  end
+
+  def to_millilitre({:tablespoon, tablespoons}) do
+    {:millilitre, tablespoons * 15}
+  end
+
+  def to_millilitre({:millilitre, _} = volume) do
+    volume
+  end
+
+  # Convert from millilitre to another unit
+
+  def from_millilitre({:millilitre, mls}, :cup) do
+    {:cup, mls / 240}
+  end
+
+  def from_millilitre({:millilitre, mls}, :fluid_ounce) do
+    {:fluid_ounce, mls / 29.57}
+  end
+
+  def from_millilitre({:millilitre, mls}, :teaspoon) do
+    {:teaspoon, mls / 5}
+  end
+
+  def from_millilitre({:millilitre, mls}, :tablespoon) do
+    {:tablespoon, mls / 15}
+  end
+
+  def from_millilitre({:millilitre, _} = volume, :millilitre) do
+    volume
+  end
+
+  # convert from a supported unit to a supported unit
+
+  @doc """
+  Convert the volume_pair to another unit
+  """
```

Comment:
I don't think we have an exercise for docstrings, I would probably remove this.

Reply:
Agreed.